fetch.js

```diff
@@ -0,0 +1,6 @@
+module.exports = function(url, options) {
+	if (/^\/\//.test(url)) {
+		url = 'https:' + url;
+	}
+	return require('node-fetch').call(this, url, options);
```

Comment:
I would recommend caching this require call outside of the function and ensure it closes over it. Without this change, I believe you would be hitting the synchronous require with each fetch.